selfservice/strategy/oidc/provider_userinfo_test.go

```diff
@@ -89,6 +89,16 @@ func TestProviderClaimsRespectsErrorCodes(t *testing.T) {
 				Provider:  "auth0",
 			}, reg),
 		},
+		{
+			name:             "salesforce",
+			userInfoHandler:  defaultUserinfoHandler,
+			userInfoEndpoint: "https://www.salesforce.com/userinfo",
+			provider: oidc.NewProviderAuth0(&oidc.Configuration{
```

Comment:
Is this correct?

Reply:
Damnit ... Nice catch:
https://github.com/ory/kratos/pull/4007